Stripped trailing comma from final element of enumeration to prevent compiler warning under GCC -pedantic.
Index: boost1.54-1.54.0/boost/interprocess/errors.hpp
===================================================================
--- boost1.54-1.54.0.orig/boost/interprocess/errors.hpp	2012-09-24 08:17:34.000000000 -0400
+++ boost1.54-1.54.0/boost/interprocess/errors.hpp	2013-08-05 08:57:29.171563165 -0400
@@ -114,7 +114,7 @@
    not_such_file_or_directory,
    invalid_argument,
    timeout_when_locking_error,
-   timeout_when_waiting_error,
+   timeout_when_waiting_error
 };
 
 typedef int    native_error_t;
